code/modules/surgery/organs/lungs.dm

```diff
@@ -54,8 +56,40 @@
 
 	var/crit_stabilizing_reagent = "epinephrine"
 
+	//health
+	var/maxHealth = LUNGS_MAX_HEALTH
+	var/damage = 0
+
+//TODO: lung health affects lung function
+/obj/item/organ/lungs/proc/adjustLungLoss(damage_mod, mob/living/carbon/M) //damage might be too low atm.
+	message_admins("[M] lung damage: [damage_mod], damage: [damage]")
+	if (maxHealth == "plasma")
```

Comment:
make maxhealth infinite a or a define or something for making it invulnerable, anything but hardcoded string/name checks in a variable that's supposed to (from how it's named )always be a number.

Reply:
Are string operations slow?